src/haddock/modules/topology/topoaa/defaults.cfg

```diff
@@ -14,6 +14,29 @@ dna = false
 shape = false
 cg = false
 cyclicpept = false
+nhisd = 0
```

Comment:
If you ever wish HADDOCK3 checks the parameters inside `input` as it checks for the general parameters of each module. Just let me know. Currently, I don't know what is best, if to allow flexibility because there are countless of parameters that can go here, or, if we are done, we can restrict.

Reply:
This particular one should be present in the defaults and not set to None because  used in the CNS scripts.